src/testHelper.js

```diff
@@ -62,9 +63,20 @@ export async function createIntegrationTestUser() {
  * @param {Object} user created in #createIntegrationTestUser()
  * @returns {Promise}
  */
+
```

Comment:
Is there a reason between the new line between the JS Doc and the method definition?

Reply:
I really don’t care lol

Reply:
Whatever you want

Reply:
Can you remove it? I think it might have been accidental

Reply:
Let me get home. Can you approve? I will address your comments.

Reply:
Don't forget to remove this, please